modules/toast/react/stories/stories.tsx

```diff
@@ -32,7 +32,7 @@ const toastData = {
 };
 
 class ToastWrapper extends React.Component<{state?: string}> {
-  public render() {
+  render() {
```

Comment:
I think I changed this for Storybook MDX issues... Not really necessary, but `public` is the default if not specified. Less syntax to parse...